lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -47,10 +47,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
       });
     }
 
-    const pageHost = url.parse(artifacts.URL.finalUrl).host;
+    const pageHost = (new URL(artifacts.URL.finalUrl)).host;
```

Comment:
nah, we're comparing host here.  a link to example.com:51 from example.com shouldn't be considered external. 
And IMO we wouldn't want an https://example.com link from http://example.com to be considered external. Not for the purposes of this audit anyway.